### ISSUE TITLE
All scheduled charts are not updated if one of the scheduled charts is deleted

### ISSUE THREAD

**Issue by contactashish13** [Codeinwp/visualizer#265]
Scenario:

1. Create 5 scheduled charts
2. Check they are being updated correctly as per schedule
3. Delete the 2nd chart
4. During next scheduled update, only the 1st chart will be updated

This is because the process exits when the 2nd chart is not found.

**Comment by contactashish13** [comment]
@rodica-andronache PR: https://github.com/Codeinwp/visualizer/pull/266

**Comment by rodica-andronache** [comment]
@rohittm please test this alone, and with the pro version and let me know how's going

**Comment by rohittm** [comment]
@contactashish13 I tried all the steps you mentioned with the WordPress.org plugin (and not your branch i.e. issue-265) in poopy.life and the schedule import still works even after deleting the second chart. Can you tell me a little more, on how to reproduce this issue?

**Comment by contactashish13** [comment]
@rohittm https://github.com/Codeinwp/visualizer/issues/265#issue-326713400 is the only way I can reproduce it. @rodica-andronache are you able to reproduce this scenario?

**Comment by rodica-andronache** [comment]
@contactashish13 @rohittm I tested the org plugin, and from what I understand, the schedule is not working properly, or I don't understand how it would be supposed to work. I created 3 pie charts with the same csv file for schedule, and have selected 1 hour interval for all of them.
I updated the csv file after 5 minutes, and the charts got updated already. Is that normal? Shouldn't the check be at 1 hour? For me, it seems it's instantly.
